profile/README.md

```diff
@@ -0,0 +1,64 @@
+
+As a group of engineers focusing on creating platforms for developers, we found ourselves
+consistently asking the same question: _how do we make the creation and management of infrastructure
+as simple as possible for our customers so that they can focus on writing code?_
+
+Simple question, not so simple to answer! When we first began talking amongst ourselves, we always
+hoped that someone would have a concrete answer, but the only thing we found was that we all had
+difficulty coming up with one. Our experience convinced us that this was a problem worth considering
+and exploring.
+
+This GitHub organization is our attempt to publicize our experiments and thoughts around this
+question. We have no answer, but we hope you'll follow along as we stumble through.
+
+## Tenets
+
+### Define Infrastructure as Code
+
+There is a reason that [the Infrastructure as Code
+movement](https://www.hashicorp.com/resources/what-is-infrastructure-as-code) has gained so much
+popularity in recent years. Ensuring that we define infrastructure in code has many advantages,
+such as:
+
+- It ensures consistency when we provision infrastructure across many environments and regions.
+- It allows the construction of reusable components that define small, single-use pieces of
+  infrastructure, which can then be composed into larger units, much like using libraries in
+  application code.
+- It enables infrastructure provisioning and ongoing management to be automated in the same way as
+  we build and release software.
+- It allows for versioning the definition of your infrastructure, potentially in
+  concert/coordination with the software running on the infrastructure.
+
+For these reasons, we reject any solution that does not allow us to generate, export, or write the
+definition of infrastructure in a form compatible with version-controlled storage.
+
+### Do Not Force Provider Choice
+
+We do not consider solutions that are inherently proprietary or useful only in the context of a
+single cloud provider since this limits their usefulness and leads to the necessity of learning
+multiple languages/frameworks.
+
+### Favor Declarative Over Imperative
+
+We favor tools that follow a declarative versus an imperative approach. Imperative tools require
+users to consider _how to achieve a desired state_, whereas declarative tools only require users to
+_describe the desired result_. For example, imperative tooling requires users to consider factors such
+as idempotence, in-place mutation, and ongoing state management. By contrast, declarative tools
+manage this for you.
+
+### Do Not Require Developers to Learn New Languages
+
+One of our goals is to allow developers to focus on honing their craft rather than learning
+new skills. As such, we favor approaches that enable developers to provision and manage their
+infrastructure either in a language of their choice or using ubiquitously available and used
+languages or formats, such as JSON or YAML.
```

Comment:
> One of our goals is to allow developers to focus on honing their craft rather than learning
new skills. 

Some food for thought: Perhaps I'm being a bit too nit-picky, but I'm not sure I totally agree, here. I tend to think learning new skills is a critical part of "honing their craft." In my mind, good solutions may require us to learn new things and I think that can often be worth the effort. Zooming out, however, I agree with the what I suspect is the true spirit of this paragraph: enabling efficiency shouldn't be too excessively arduous or cumbersome for developers (i.e. it should be a good user experience).

...am I being too pedantic? :)

Reply:
No, I don't think you're being too pedantic. The language may have unintentionally communicated the idea that application developers _shouldn't_ or _weren't capable of_ also learning to be great infrastructure engineers or equally capable Terraform coders. I re-worked it to make it clear that the goal here is to not _force_ folks into splitting their brainpower between being great app developers _and_ managing low-level primitives of infrastructure, in the service of reducing stress an burnout.